commit 0f7f13772d91ca082342a6f20908eb6672d6c8d6 (refs/changes/58/358/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-08-23T19:38:41+00:00 (3 years, 2 months ago)
    
    FWAPI-246 fwadm(1M) should use the REQ_ID environment variable when present

diff --git a/src/fw/lib/util/log.js b/src/fw/lib/util/log.js
index cad4f7fa..f652f59e 100644
--- a/src/fw/lib/util/log.js
+++ b/src/fw/lib/util/log.js
@@ -328,7 +328,7 @@ function createLogger(opts, readOnly) {
 
     log = bunyan.createLogger({
         name: logName,
-        req_id: opts.req_id || mod_uuid.v4(),
+        req_id: opts.req_id || process.env.REQ_ID || mod_uuid.v4(),
         serializers: serializers,
         streams: streams
     });
